src/lib/gundb/UserStorageClass.js

```diff
@@ -729,6 +729,8 @@ export class UserStorage {
     const userProperties = await this.userProperties.getAll()
     const firstVisitAppDate = userProperties.firstVisitApp
 
+    this.addBackupCard()
```

Comment:
When you first start the application, the user’s properties do not yet have the property (firstVisitApp), it is set in the condition below, and the condition (firstVisitApp) is not installed by default.

Reply:
the condition inside addbackup card checks if firstVisitApp is set. if not it will not add the backupcard

Reply:
Ok, I understand